### PR TITLE
Read custom metadata needs_background property

### PR DESCRIPTION
**Pull request by joyceerhl**
For https://github.com/microsoft/vscode-jupyter/issues/5353 

Tested by copying built renderers.js from vscode-notebook-renderers to vscode-jupyter/out/client_renderer, building and debugging the Jupyter extension in the extension host, and running a code cell which generates output with the image/png mimetype in dark mode. In the screenshot below backgrounds are now white again. 

image

### CODE REVIEW

File: src/client/render.tsx [hunk 1]
```diff
@@ -50,9 +50,10 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         const imgStyle: Record<string, string | number> = {};
         const divStyle: Record<string, string | number> = { overflow: 'scroll' }; // This is the default style used by Jupyter lab.
         const imgSrc = `data:${mimeType};base64,${mimeBundle[mimeType]}`;
+        const customMetadata = metadata.metadata as JSONObject | undefined;
```
Comment (IanMatthewHuff) [inline src/client/render.tsx:53]: Ahh, looks like it was from the metadata.metadata change. Thanks for the fix!

**Review by IanMatthewHuff — APPROVED** [review]

File: src/client/render.tsx (hunk 1 above)
Comment (joyceerhl) [reply src/client/render.tsx:53]: I remembered you mentioning this at triage, so it was the first thing I tried 😊 thanks for the pointer!

**Review by rchiodo — APPROVED** [review]

File: src/client/render.tsx (hunk 1 above)
Comment (IanMatthewHuff) [reply src/client/render.tsx:53]: I mean...I caused the bug, you shouldn't be thanking me 😆 . Just didn't realize it when I made the change. I checked widgets, but didn't notice the background regression.

**Review by DonJayamanne — APPROVED** [review]

**Comment by DonJayamanne** [comment]
Might want to address the CI issue seprately, as a separate PR

**Comment by joyceerhl** [comment]
@donjayamanne yes I agree 😅 

AzDO status: 
image

**Comment by joyceerhl** [comment]
Would someone mind merging this into main please? I don't have permissions to merge it.

**Comment by joyceerhl** [comment]
@rchiodo thank you!

**Pull request merged**